src/screens/HomeScreen/components/NameBarContainer.js

```diff
@@ -46,7 +54,9 @@ const NameBarContainer = () => {
       />
       <Button title="Clear word list" onPress={handleClearButtonPress} />
       <ScrollView>
-        <Text>User Data: {JSON.stringify(userData)}</Text>
+        <Text onPress={refreshList}>
```

Comment:
It may be worth investigating whether there are accessibility concerns to adding an `onPress` handler to a `Text` node. I don't remember off the top of my head whether that creates any issues, but it's something to look into.

Reply:
Yeah that makes sense, I would imagine there are. In this case, this is something that I'm just using to help me develop and plan to take out of the final product